src/js/profile/charts/barChart.js

```diff
@@ -223,7 +227,7 @@ export const configureBarchart = (data, metadata, config) => {
     };
 };
 
-export const configureBarchartDownload = (data, metadata, config, annotations) => {
+export const configureBarchartDownload = (data, filteredData,metadata, config, annotations) => {
```

Comment:
I don't understand why we need to pass data and filteredData? That seems like we're handling variables with duplicated data. Is it possible to only parse data or filteredData

Reply:
@michaelglenister actually we are not running filters on configureBarchartDownload view so we will have to pass

* `data` to get all domains (y-axis labels)
* `filtered data` to get actual filtered data from chart vega view